### PR TITLE
Improve dockerization

### PR DESCRIPTION
**Pull request by jwroblewski**

### CODE REVIEW

**Comment by jwroblewski** [comment]
[Dockerized full stack dev setup for testing](https://trello.com/c/Vu2doJNj/1514-dockerized-full-stack-dev-setup-for-testing)

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Dockerfile
```
Comment (nightlyone) [inline .dockerignore:1]: If the Dockerfile changes, a change to the docker image is likely, too. So my personal recommendation would be to remove this line.

Comment (jwroblewski) [reply .dockerignore:1]: If a Dockerfile change affects the image, then the image will get rebuilt by the successive `docker build` anyway. The motivation here was not to trigger unneeded builds in the case when changes to Dockerfile do not affect the image content, eg formatting adjustments. Such changes would normally invalidate a cached `COPY . .` layer and trigger a rebuild of successive layers, which can take some time if they happen to involve `go get` or `bundle install`.

**Comment by danchel** [comment]
:+1:

**Pull request merged**